src/index.js

```diff
@@ -32,9 +32,14 @@ const retrieveVisitorToken = (config = {}) => {
   return token;
 };
 
-const retrievePageUUIDToken = () => (
-  Cookie.get(PAGE_UUID) || uuid()
-);
+const retrievePageUUIDToken = () => {
+  let token = Cookie.get(PAGE_UUID);
+  if (!token) {
+    token = uuid();
+    Cookie.set(PAGE_UUID, token);
+  }
+  return token;
+};
```

Comment:
this test should take care of setting up cookie https://github.com/clutter/wt-client/blob/master/test/index.js#L281

Reply:
that just looks at the wt event payload, not the cookie. it's important to check for the cookie specifically because we rely on it being set when we look for it on the server side.

Reply:
we will not be able to test cookie is set properly and we need to introduce a new plugin mock-browser